src/components/OrgPostCard/OrgPostCard.tsx

```
@@ -481,21 +553,18 @@
+                {localPreviews
+                  .filter((preview) => preview.mimeType.startsWith('video/'))
+                  .map((preview, index) => (
                     <div key={index} className={styles.previewOrgPostCard}>
                       <video controls data-testid="video-preview">
-                        <source
-                          src={attachment.url}
-                          type={attachment.mimeType}
-                        />
+                        <source src={preview.url} type={preview.mimeType} />
                         {t('videoNotSupported')}
                       </video>
```

Biome (2.1.2)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)